x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/derivative.tsx

```diff
@@ -95,5 +99,13 @@ export const derivativeOperation: OperationDefinition<
       })
     );
   },
+  getDisabledStatus(indexPattern, layer) {
+    return checkForDateHistogram(
+      layer,
+      i18n.translate('xpack.lens.indexPattern.movingAverage', {
+        defaultMessage: 'Moving average',
```

Comment:
Those messages all reference moving average, but I guess they are meant to reference the respective calculation (in this case "Differences")